views/layouts/main.handlebars

```diff
@@ -8,6 +8,9 @@
 </head>
 <body>
     <h1>OMG</h1>
+    <nav>
```

Comment:
This isn't needed the stuff from the profile page will be loaded directly into the {{{body}}}